client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -21,7 +21,7 @@ import './style.scss';
 
 interface CustomerHomeLaunchpadProps {
 	checklistSlug: string;
-	extraActions?: PermittedActions;
+	extraActions?: Omit< PermittedActions, 'setActiveChecklist' >;
```

Comment:
This isn't quite what I was thinking of in [my original comment](https://github.com/Automattic/wp-calypso/pull/81576#discussion_r1329111725) - I was wondering whether we want the wiring for `setActiveChecklist` to be implemented in the launchpad package itself. I think that would introduce a circular dependency, however, so maybe it's better just to keep this approach as-is.